lib/active_record/mysql/enum/validations.rb

```diff
@@ -13,18 +13,18 @@ def validates_columns(*column_names)
           column_names.each do |name|
             col = cols[name.to_s]
             raise ArgumentError, "Cannot find column #{name}" unless col
-            
+
             # test for nullability
             validates_presence_of(name) if !col.null
-            
+
             # Test various known types.
             case col.type
             when :enum
               validates_inclusion_of name, :in => col.limit, :allow_nil => true
-              
+
             when :integer, :float
               validates_numericality_of name, :allow_nil => true
```

Comment:
Huh, these other `allow_nil: true` settings look wrong too. Weird.

Reply:
Agreed, are you comfortable leaving as is for now while we get this in use and improve when we have unit tests up and running?

Reply:
Sure. And per my later comment above, in the next pass, I think we should remove this buggy code that doesn't involve enums!